### ISSUE TITLE
Evaluate and update the focus-state styles for all interactive elements (including custom elements and form elements)

### ISSUE THREAD

**Issue by blnkt** [ManifoldScholar/manifold#1239]
This should be addressed in the frontend section first, then the reader, and finally the backend admin section. Go to each route (maybe work through the routes file in order to make sure you visit all the top-level views) and tab through every focusable element. Does the focus state of each element look OK? Can we improve it with styles? Refer to the following style guide for some updated button state styles.  Use hoverstate if focus state is not provided.  Use best judgement where existing design or hoverstate is not available.

- [x]  Frontend
- [ ]  Reader
- [ ]  Admin Backend

[Buttons - Frontend.pdf](https://github.com/ManifoldScholar/manifold/files/2220442/Buttons.-.Frontend.pdf)

Do this work in this branch/PR: https://github.com/ManifoldScholar/manifold/pull/1116/commits/66667aa480e4e2356b05d8a3e3a8c3bef5ab3b92
